Components/GameBoard/PlayerStats.jsx

```diff
@@ -49,7 +49,7 @@ export class PlayerStats extends React.Component {
     render() {
         var playerAvatar = (
             <div className='player-avatar'>
-                <Avatar emailHash={ this.props.user ? this.props.user.emailHash : 'unknown' } />
+                <Avatar username={ this.props.user ? this.props.user.username : 'unknown' } />
```

Comment:
If we have someone with username `'unknown'`, they won't be able to have an avatar under this scheme.